app/lib/RouteManager.js

```diff
@@ -45,7 +45,7 @@ module.exports = function RouteManager(app) {
 
     record: function record(captureName) {
       const capture = new CaptureRecorder(app, captureName);
-      this.register('all', '*', capture.record.bind(capture));
+      app.use(capture.record.bind(capture));
```

Comment:
It was actually more complex to pipe this through the route registration process, and I think it will behave the same as its own middleware?

Reply:
I expect this to be functionally equivalent.